### PR TITLE
Add flush command

### PR DESCRIPTION
**Pull request by wHoIsDReAmer**
## Chnages
* Add flush command
* Reduce binary file size

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -6,6 +6,12 @@ license = "MIT"
 repository = "https://lab.themackabu.dev/self/pmc"
 description = "PMC is a simple and easy to use PM2 alternative"
 
+[profile.release]
+lto = true
+codegen-units = 1
+opt-level = "z"
```
Comment (theMackabu) [inline Cargo.toml:12]: I personally had build issues with codegen level 1, not sure if I should merge this in with this PR.

File: src/cli/internal.rs [hunk 2]
```diff
@@ -161,6 +161,12 @@ impl<'i> Internal<'i> {
         log!("process removed (id={})", self.id);
     }
 
+    pub fn flush(&mut self) {
+        self.runner.flush(self.id);
```
Comment (theMackabu) [inline src/cli/internal.rs:165]: Can add a temporary program stop here when trying to use a external server, as the code for flushing external logs was not added in this PR, and could confuse users.

**Review by theMackabu — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (wHoIsDReAmer) [reply Cargo.toml:12]: I see

**Review by theMackabu — APPROVED** [review]

**Comment by DevilOfLegend** [comment]
I love you boy

**Pull request merged**